pyproject.toml

```diff
@@ -9,9 +9,12 @@ python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
 requests = "^2.25.1"
+pytest = "^6.2.2"
```

Comment:
We don't need the dependencies to be listed in both the dependencies and the dev-dependencies sections - we can have them solely in the list below